testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java

```diff
@@ -276,9 +276,9 @@ public void onAcquireTokenSilentWithResourceClicked(final AcquireTokenFragment.R
         prepareRequestParameters(requestOptions);
 
         //final IAccount requestAccount = getAccount();
-        mApplication.getAccounts(new PublicClientApplication.AccountsLoadedCallback() {
+        mApplication.getAccounts(new IAccountCallback<List<IAccount>>() {
```

Comment:
Is IAccountCallback part of the public API of MSAL?  I wouldn't have expected that.  Isn't this an internal implementation in common?

Reply:
Addressed. Thanks